### DISCUSSION TITLE
Desktop Not Seeing Running VM on MacOS

### DISCUSSION THREAD

**Discussion by azizabah** [podman-desktop/podman-desktop#170] [Q&A] [answered]
So I have been using podman CLI for a while to replace docker and was hoping to utilize the desktop to simplify things.

Running this command in terminal (MacOS Intel):
$ podman machine list
NAME                     VM TYPE     CREATED      LAST UP            CPUS        MEMORY      DISK SIZE
podman-machine-default*  qemu        7 weeks ago  Currently running  1           2.147GB     53.69GB

But inside the UI, it doesn't think a container engine is running? Is there something else that needs to happen for it to see it. Apologies if this discussion isn't the right place.

**Comment by benoitf** [comment]
@azizabah it's the right place.

Could you tell me the version of podman that you're using ?

using `podman version` command

**Reply by azizabah** [reply]
$ podman version
Client:
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.17.3
Built:        Wed Dec  8 13:41:11 2021
OS/Arch:      darwin/amd64

Server:
Version:      4.1.0
API Version:  4.1.0
Go Version:   go1.18
Built:        Fri May  6 12:15:54 2022
OS/Arch:      linux/amd64

**Comment by benoitf** [comment, marked as answer]
@azizabah ok so here the issue is that podman 3.x is not providing a socket on the host.
So Podman Desktop is not able to connect through this socket.

Could you please update the client version to be 4.1.0 as well ? (it seems your server is up-to-date but not the client side)

v4.0 is required on macOS or on Windows

**Reply by azizabah** [reply]
That did it. Thanks @benoitf !

**Reply by benoitf** [reply]
we want to add some preflight checks to detect such "bad behaviour" of Podman Desktop and provide a better guidance/feedback to users.
